epictrack-web/src/components/indigenousNation/IndigenousNationList.tsx

```diff
@@ -216,12 +216,8 @@ export default function IndigenousNationList() {
         },
         Cell: ({ cell }) => (
           <span>
-            {cell.getValue<boolean>() && (
-              <ActiveChip label="Active" color="primary" />
-            )}
-            {!cell.getValue<boolean>() && (
-              <InactiveChip label="Inactive" color="error" />
-            )}
+            {cell.getValue<boolean>() && <ETChip active label="Active" />}

```

Comment:
What do you think of using a variant prop like in MUI. The boolean will only work for two cases active or inactive. A variant that takes a string "active", "inactive", "somethingelse" would allow us to add more chip variants